bitwarden_license/bit-web/src/app/secrets-manager/guards/sm-org-enabled.guard.ts

```diff
@@ -1,40 +1,23 @@
 import { inject } from "@angular/core";
-import {
-  ActivatedRouteSnapshot,
-  CanActivateFn,
-  createUrlTreeFromSnapshot,
-  RouterStateSnapshot,
-} from "@angular/router";
+import { ActivatedRouteSnapshot, CanActivateFn, createUrlTreeFromSnapshot } from "@angular/router";
 
-import { AuthGuard } from "@bitwarden/angular/auth/guards";
 import { OrganizationService } from "@bitwarden/common/admin-console/abstractions/organization/organization.service.abstraction";
-import { AuthService } from "@bitwarden/common/auth/abstractions/auth.service";
-import { AuthenticationStatus } from "@bitwarden/common/auth/enums/authentication-status";
 import { SyncService } from "@bitwarden/common/vault/abstractions/sync/sync.service.abstraction";
 
 /**
  * Redirects from root `/sm` to first organization with access to SM
  */
-export const organizationEnabledGuard: CanActivateFn = async (
-  route: ActivatedRouteSnapshot,
-  state: RouterStateSnapshot
-) => {
+export const organizationEnabledGuard: CanActivateFn = async (route: ActivatedRouteSnapshot) => {
   const syncService = inject(SyncService);
-  const authService = inject(AuthService);
   const orgService = inject(OrganizationService);
-  const authGuard = inject(AuthGuard);
 
   /** Workaround to avoid service initialization race condition. */
   if ((await syncService.getLastSync()) == null) {
     await syncService.fullSync(false);
   }
 
-  if ((await authService.getAuthStatus()) !== AuthenticationStatus.Unlocked) {
-    return authGuard.canActivate(route, state);
-  }
-
   const org = orgService.get(route.params.organizationId);
-  if (org == null) {
+  if (org == null || !org?.canAccessSecretsManager) {
```

Comment:
Nit: you don't need the `?` because you already test if `org` is null.

```suggestion
  if (org == null || !org.canAccessSecretsManager) {
```